Block/Adminhtml/System/Config/Fieldset/Payment.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+namespace Amazon\Pay\Block\Adminhtml\System\Config\Fieldset;
+
+use Amazon\Pay\Model\Client;
+use Amazon\Pay\Model\Serializer;
+use Magento\Backend\Block\Context;
+use Magento\Backend\Model\Auth\Session;
+use Magento\Config\Model\Config;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Exception\FileSystemException;
+use Magento\Framework\Filesystem\Driver\File;
+use Magento\Framework\View\Helper\Js;
+use Magento\Framework\View\Helper\SecureHtmlRenderer;
+use Magento\Framework\Module\Dir\Reader;
+
+class Payment extends \Magento\Paypal\Block\Adminhtml\System\Config\Fieldset\Payment
+{
+
+    const MODULE_CODE = 'Amazon_Pay';
+    const MODULE_PACKAGE_NAME = 'amzn/amazon-pay-magento-2-module';
+    const MODULE_COMPOSER_FILE = '/composer.json';
+    const MODULE_CHANGELOG_URL = 'https://github.com/amzn/amazon-payments-magento-2-plugin/blob/master/CHANGELOG.md';
+    /**
+     * @var SecureHtmlRenderer
+     */
+    private $secureRenderer;
+
+    /**
+     * @var Reader
+     */
+    private $moduleReader;
+
+    /**
+     * @var File
+     */
+    private $filesystem;
+
+    /**
+     * @var Serializer
+     */
+    private $serializer;
+
+    /**
+     * @var Client
+     */
+    private $client;
+
+    public function __construct(
+        Context $context,
+        Session $authSession,
+        Js $jsHelper,
+        Config $backendConfig,
+        Reader $moduleReader,
+        File $filesystem,
+        Serializer $serializer,
+        Client $client,
+        SecureHtmlRenderer $secureHtmlRenderer,
+        array $data = [],
+
```

Comment:
Rather ticky-tack, but the trailing comma in the argument list here breaks backward incompatibility with PHP 7.x.